pkgs/development/interpreters/ruby/patchsets.nix

```diff
@@ -16,4 +16,6 @@ rec {
     "${patchSet}/patches/ruby/2.5/head/railsexpress/02-improve-gc-stats.patch"
     "${patchSet}/patches/ruby/2.5/head/railsexpress/03-more-detailed-stacktrace.patch"
   ];
+  "2.6.0" = ops useRailsExpress [
+  ];
```

Comment:
Presumably this is here because Rails express patches aren’t available yet? If that’s the case, could we have a comment to that effect explaining the empty code?

Reply:
I added a comment
FWIW I'm testing ruby 2.6 with a mid-size rails application and I haven't found any downside